src/CredentialManager.js

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright © 2024 Twilio Inc.
```

Comment:
Is this included in all the files? Even the sample app?

Reply:
Yes, this is included in all files, except for those that works as config files, like `rollup.config.js` or `tsconfig.json`, this should be included on those files? also not in sample-app due to was added in a parallel PR

Reply:
Already added to the sample-app